src/games/strategy/triplea/ai/proAI/util/ProMoveUtils.java

```diff
@@ -271,6 +272,39 @@ public static void calculateBombardMoveRoutes(final PlayerID player, final List<
     }
   }
 
+  public static void calculateBombingRoutes(final PlayerID player, final List<Collection<Unit>> moveUnits,
```

Comment:
Move this entire method into that class as well? Then it'll have some meat to it